Cargo.toml

```diff
@@ -1,10 +1,12 @@
 [patch.crates-io]
```

Comment:
If we go along the simple, single file route, these changes will not be necessary